index.html

```diff
@@ -23,25 +23,162 @@
     </header>
 
 
-    <section class="container">
+    <section id="about">
 
-    <h1 class="primary-text">
+    <h1 class="title">
             I’m Miguel Cuello.
             Glad to see you!
-        </h1>
-        <p class="first-text">
+    </h1>
+        <p class="text">
             I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
         </p>
-        <h3>LET’S CONNECT</h3>
-        <ul id="social">
-            <li><a href="#"><img src="images/twitter.svg" alt="twitter_logo" id="twitter" class="icon"></a></li>
-            <li><a href="#"><img src="images/linkedin.svg" alt="linkedin_logo" id="linkedin" class="icon"></a></li>
-            <li><a href="#"><img src="images/M.svg" alt="capital_m" id="m" class="icon"></a></li>
-            <li><a href="#"><img src="images/git.svg" alt="github_logo" id="github" class="icon"></a></li>
-            <li><a href="#"><img src="images/hand.svg" alt="hand" id="hand" class="icon"></a></li>
+
+    <h3>LET’S CONNECT</h3>
+
+    <ul class="social">
+        <li><a href="#"><img src="images/twitter.svg" alt="twitter_logo" id="twitter" class="icon"></a></li>
+        <li><a href="#"><img src="images/linkedin.svg" alt="linkedin_logo" id="linkedin" class="icon"></a></li>
+         <li><a href="#"><img src="images/hand.svg" alt="hand" id="hand" class="icon"></a></li>
+        <li><a href="#"><img src="images/git.svg" alt="github_logo" id="github" class="icon"></a></li>
+        <li><a href="#"><img src="images/M.svg" alt="capital_m" id="m" class="icon"></a></li>
+    </ul>
+
+    </section>
+
+    <section  class="card" id="tonic">
+
+        <img src="images/portfolio.png" alt="a_screentshot" id="portfolio">
+
+        <h1 class="title tonic">Tonic</h1>
+
+        <img src="images/fame2.png" alt="frame" id="frame">
+
+        <p class="tonic-p">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="code-links">
+            <li class="tag">
+                <a href="" alt="">HTML</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">CSS</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">JavaScript</a>
+            </li>
         </ul>
 
+        <h3 class="button"><a href="" id="see_project">See Project</a></h3>
+
     </section>
+
+    <section id="multi" class="card">
+
+        <img src="images/portfolio2.png" alt="a_screentshot" id="portfolio">
+
+        <h1 class="title tonic">Multi-Post Stories</h1>
```

Comment:
- [ ] Why are you not considered to be using other heading elements `h2` .... `h6` since you have already used `h1` in the header tag and it is recommended to use the `h1` element once in an HTML document. :blush: